front/src/views/profile/eating/eating.view.html

```diff
@@ -106,6 +106,16 @@ <h6 :title="place.serverInfo.placeName" class="text-semi-bold d-flex justify-sta
       </GmapMap>
     </v-col>
   </v-row>
+  <v-pagination
+    circle
+    color="primary"
+    totalVisible="5"
+    :length="totalPageCount+1"
```

Comment:
I think, pageCount should be correct without any math operations.
Or you should name it maxPageIndex  :)

Reply:
ok